plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestS3WrongRegionPicked.java

```diff
@@ -27,12 +27,14 @@
     public void testS3WrongRegionSelection()
             throws Exception
     {
-        try (HiveMinioDataLake dataLake = new HiveMinioDataLake("test-bucket").start();
+        String bucketName = "test-bucket" + randomTableSuffix();
+
+        try (HiveMinioDataLake dataLake = new HiveMinioDataLake(bucketName).start();
```

Comment:
I'm not quite sure if this autoclosable is stopping containers. It stops MinoClient.